src/main/java/com/opinta/mapper/ClientMapper.java

```diff
@@ -5,8 +5,9 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.mapstruct.Mappings;
+import org.springframework.stereotype.Component;
 
-
+@Component
```

Comment:
Are you sure it doesn't work without this annotation?
It worked before why do we need it now?